rancher/rancher
# Fleet + snapshot & restore tests

## Issue: 

https://github.com/rancher/qa-tasks/issues/1525
 
## Problem

adding fleet tests where snapshots are taken in the process
 
## Solution

* move etcd snapshot helpers -> actions
* add folder for snapshots in fleet (validation/fleet/snapshot)
 
## Testing

tested on rke1 and rke2 for snapshot only
tested on rke2 for fleet + snapshots (rke1 not supported due to cluster checks in setup, can add later if need be)


## QA Testing Considerations

 https://github.com/rancher/shepherd/pull/304
see notes in ^ for some improvements in shepherd and some other considerations
